src/Uno.Extensions.Navigation.UI/FrameworkElementExtensions.cs

```diff
@@ -55,12 +54,18 @@ public static Task HostAsync(
 			{
 				start = () => nav.NavigateViewModelAsync(root, initialViewModel);
 			}
+			else
```

Comment:
I would have thought that this would follow the logic of the previous two conditions that performs the navigation if `initialRoute` is not null